client/elife-theme/src/fonts/index.css

```diff
@@ -1,16 +1,24 @@
-/* Demi */
+/* SemiBold */
 @font-face {
-  font-family: 'Avenir Next Webfont';
-  font-weight: 700;
+  font-display: fallback;
```

Comment:
Are you sure this does what you want? Setting `font-display` to `fallback` will set the invisible & swap periods for font to be small (e.g. 100ms). What this means is: first the browser will hide the font for a short period of time (invisible) while it tries to download that font. Then, if it wasn't able to download it, it will try to use a fallback font from the element itself (swap period) before the actual font is loaded. If that fails too, it will load the next system font from the font stack.

For more info
https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display
https://css-tricks.com/font-display-masses/

From my understanding you cannot specify a fallback font here. You have to do it at variable/css level.

Reply:
Looking at the variables, the fallback fonts are the ones specified after `"Noto Sans"`, e.g. for `fontInterface` it will try to load `Arial` if `Noto Sans` fails, and if `Arial` fails then `Helvetica` etc.

Reply:
`font-display: fallback` is how the eLife site is currently configured for eLife, and so xPub should have the same value. If we change the eLife behaviour, I can let @hdrury1 know that this should be updated here.

These font stacks seem correct: https://github.com/elifesciences/elife-xpub/pull/233/files/ee01aef5b81f06e9278c8a36d4c4401ece0d3b48#diff-b997819fc7f534ccbc7ed0f02dc3aa50R22

Reply:
Ok, cool :)